plugin/src/main/java/io/jenkins/plugins/analysis/warnings/Grype.java

```diff
@@ -0,0 +1,31 @@
+package io.jenkins.plugins.analysis.warnings;
+
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.jenkinsci.Symbol;
+import hudson.Extension;
+import io.jenkins.plugins.analysis.core.model.AnalysisModelParser;
+
+/**
+ * Provides parser for grype reports.
+ */
+public class Grype extends AnalysisModelParser {
+    private static final long serialVersionUID = -7721519870683487886L;
+    private static final String ID = "grypescanner";
```

Comment:
This ID is used also as URL. So if you want it to be shorter (e.g. "grype") then we need to change it in analysis-model.

Reply:
I have changed id to "grype" in [PR 936](https://github.com/jenkinsci/analysis-model/pull/936).  I will update the ID in the current PR once the change in in [PR 936](https://github.com/jenkinsci/analysis-model/pull/936) is released. After that, I will also update the new version of the model API, 11.5.1, here. Thanks!

Reply:
As discussed, id is changed to "grype", API dependency set to 11.5.1. Ready for new version of warnings-ng-plugin :-)